doc/assessment.md

```diff
@@ -66,9 +95,9 @@ An assessment is broken down in to a list of zones, like this:
             {"id": "reallyHardQ", "points": [10, 10, 10]},
             {
                 "numberChoose": 1,
-                "points": 5,
+                "points": [5, 3, 1],
```

Comment:
I think this example should switch showing the zone implementation instead of the alternative question list, which is covered below. Maybe update it to:

```json
"zones": [
    {
        "title": "True and False",
        "questions": [
            {"id": "TFMathSymbols", "points": [1]},
            {"id": "TFQuadratic", "points": [3]},
            {"id": "TFGit", "points": [1, 0.5]}
        ]
    },
{			
        "title": "Multiple Choice",
        "questions": [
            {"id": "MCDataTypes", "points": [2, 1]},
            {"id": "MCVariableNames", "points": [2, 1]},
            {"id": "MCAtomicData", "points": [2, 1]},
            {"id": "MCExecuteCode", "points": [2, 2, 2, 1, 1, 0.5]}
        ]
    }
]
```

Reply:
Good thinking, done!